lib/python/Plugins/Extensions/FileCommander/plugin.py

```diff
@@ -936,8 +936,8 @@ def keyGoBottom(self):
 	def keyHashes(self, path=None):
 		if path is None:
 			path = self.sourceColumn.getPath()
-		if isfile(path):
-			if stat(path).st_size < HASH_CHECK_SIZE:
+		if isfile(path) or islink(path):
+			if lstat(path).st_size < HASH_CHECK_SIZE:
```

Comment:
So you want to get the has from the target and not from the link?
Then you need to do more things.